bridge_adaptivity/config/settings/base.py

```diff
@@ -192,3 +193,14 @@
 
 # Demo functionality
 TEST_SEQUENCE_SUFFIX = 'test_sequence_suffix'
+
+ASGI_APPLICATION = "config.routing.application"
+
+CHANNEL_LAYERS = {
+    "default": {
+        "BACKEND": "channels_redis.core.RedisChannelLayer",
+        "CONFIG": {
+            "hosts": [("redis", 6379)],
+        },
+    },
+}
```

Comment:
A good practice is to add a trailing empty line at the end of a file.

Reply:
I added the new line.